metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -683,3 +689,19 @@ def generate_access_token(
     return token_name, response.json().get("data", {}).get("createAccessToken", {}).get(
         "accessToken", None
     )
+
+
+def ensure_mce_has_system_metadata(mce: MetadataChangeEvent) -> None:
+    if mce.systemMetadata is None:
+        mce.systemMetadata = SystemMetadataClass()
+    mce.systemMetadata.clientId = datahub.__package_name__
+    mce.systemMetadata.clientVersion = datahub.__version__
+
+
+def ensure_mcp_has_system_metadata(
+    mcp: Union[MetadataChangeProposal, MetadataChangeProposalWrapper]
+) -> None:
```

Comment:
let's put this in a separate utility file?

also, we don't need two methods - let's just have a `inject_client_version_system_metadata` or something that can handle both MCPs and MCEs

Reply:
Made a single method. Do you have a preferred name for the separate file? If you can suggest a name that will be ok for you to approve do let me know and I'll move to using that.